### PR TITLE
Bleeerg post

### PR DESCRIPTION
**Pull request by bcardarella**
best practices for the embers

### CODE REVIEW

File: source/posts/2015-10-19-2015-dont-dont-override-init.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+baz: on('init', () => {
+  // ...
+})
+```
+
+After:
+
+```js
+init() {
+  foo();
```
Comment (rwjblue) [inline source/posts/2015-10-19-2015-dont-dont-override-init.md]:
```
this.foo();
this.baz();
this.bar();
```

Comment (rwjblue) [inline source/posts/2015-10-19-2015-dont-dont-override-init.md]: Also, `this._super` should **always** be first (it is a requirement in ES6 classes to call `super` before touching `this`).

**Comment by rwjblue** [comment]
One thing to note, this also applies to almost all other hooks. I have seen things like `on('didInsertElement` quite a bit, and I think the method approach is much better.

**Comment by duggiefresh** [comment]
Yeah, I like using the new component hooks directly without `on`. I think that sounds like a bleeeeerg post!

**Pull request merged**